front/lib/api/assistant/configuration.ts

```diff
@@ -56,12 +57,9 @@ import { generateModelSId } from "@app/lib/utils";
  * Get an agent configuration
  *
  */
-export async function getAgentConfiguration(
+export async function getAgentConfigurationDetailedView(
```

Comment:
Same comment from a different angle.

If we call it DetailedView feels weird here since the type is still `AgentConfiguration` 

Pushing for an alignment everywhere.